app/src/main/java/com/vero/tasky/agenda/data/util/NotificationBuilder.kt

```diff
@@ -0,0 +1,38 @@
+package com.vero.tasky.agenda.data.util
+
+import android.app.Notification
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import com.vero.tasky.MainActivity
+import com.vero.tasky.R
+import com.vero.tasky.agenda.domain.remindermanager.ReminderManager
+
+object NotificationBuilder {
+
+    fun buildReminderNotification(
```

Comment:
Same here
Reminder is different that agenda item reminder

So id find a less confusing name